pkg/storaged/fsys-tab.jsx

```diff
@@ -549,6 +599,10 @@
             }
             if (opt_ro)
                 opts.push("ro");
+            if (opt_nofail)
+                opts.push("nofail");
+            if (opt_netdev)
+                opts.push("netdev");
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-17888-20221124-142402-f4c856ed-fedora-36-devel//Coverage/lcov/github-pr.diff.gcov.html)